components/identity/module/IdentityPopover.vue

```diff
@@ -26,6 +26,10 @@ import { useIdentitySoldData } from '../utils/useIdentity'
 import IdentityPopoverHeader from './IdentityPopoverHeader.vue'
 import IdentityPopoverFooter from './IdentityPopoverFooter.vue'
 
+const props = defineProps<{
+  size?: 'small' | 'medium'
+}>()
+
```

Comment:
WHY? It seems not related to the size. 

If you want to control its shown, you should use another proper name.